packaging/docker/custom/ubuntu.Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GRAFANA_VERSION="latest-ubuntu"
+ARG GRAFANA_VERSION="latest"

```

Comment:
Why this change?

Reply:
Because of https://github.com/grafana/grafana/pull/21837/files#diff-4dccfdd8f9f661298752023ca816565fR3

Reply:
Bugfix

Reply:
Ah